services/ux-backend/handlers/cleanupconsumer/handler.go

```diff
@@ -0,0 +1,83 @@
+package cleanupconsumer
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	ocsv1alpha1 "github.com/red-hat-storage/ocs-operator/api/v4/v1alpha1"
+	"github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func HandleMessage(w http.ResponseWriter, r *http.Request, cl client.Client) {
+	switch r.Method {
+	case "DELETE":
+		consumerName := r.URL.Query().Get("consumerName")
```

Comment:
consumer name should come from the path not from the query. 
The resource we are trying to delete is: `/sotrageconsumers/<consumer_id>`